### DISCUSSION TITLE
4-space tab warning

### DISCUSSION THREAD

**Discussion by qbatten** [vslinko/obsidian-outliner#183] [Archive]
Firstly, this plugin is AWESOME, thank you for your work!

I'm trying to understand the warning about needing to use 4-space tabs. 

Is the plugin checking some setting in Obsidian and popping up that warning in the status bar if the setting is incorrect? Or is it checking the current file to see if it has the wrong tab-space-count? Or is it just always there until you manually turn it off?

I think i have the right settings for 4-space tabs, but the warning still shows, even after restarting the app and turning the plugin on and off. I have `Editor > Use tabs = False` and `Editor > Tab size = 4`. Is that the right setup for this plugin?

Thanks!

**Comment by vslinko** [comment]
@qbatten Hello!

It checks the Obsidian settings for compatibility purposes.
If you want to have all style enhancements, you need to use `Use tabs = true` and `Tab size = 4`.
If you want to use different tabs settings, you could disable the warning in the plugin settings.

**Reply by qbatten** [reply]
Aha... I got the tabSize = 4 but didn't realize it needed to be tabs = true as well. Thank you!

**Comment by miluoshi** [comment]
I've created a PR that adds support for `Use tabs = false` as well: https://github.com/vslinko/obsidian-outliner/pull/202

**Reply by qbatten** [reply]
Awesome, this is a neat way to handle it!
